front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -143,27 +215,40 @@ const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchS
             clearButton
             noMargin
             sm
-            focus
```

Comment:
the focus was added in a recent PR (https://github.com/osrd-project/osrd/pull/6157), it should be kept

Reply:
Yes, for the moment I've removed it, because with the new input I've added, there's a problem of loss of focus on it. In fact, when you start writing in this second input, the focus is instantly taken back by the first input. 
The problem may lie in the way focus is handled in inputSNCF. But the problem has been solved by [@SarahBellaha  PR](https://github.com/osrd-project/osrd/pull/6204). So, to avoid conflicts, I preferred to let the problem resolve itself naturally with her PR.